parsers/to-css-custom-properties/README.md

```diff
@@ -27,14 +28,14 @@ interface parser {
 
 ### Options
 
-| Parameter                | Required | Type                                                              | Default     | Description                                                                    |
-| ------------------------ | -------- | ----------------------------------------------------------------- | ----------- | ------------------------------------------------------------------------------ |
-| `formatName`             | optional | `camelCase` `kebabCase` `snakeCase` `pascalCase`                  | `kebabCase` | The case transformation you want to apply to your design token name            |
-| `formatTokens.color`     | optional | `rgb` `prgb` `hex` `hex6` `hex3` `hex4` `hex8` `name` `hsl` `hsv` | `rgb`       | The color format you want to apply to your potential color design token        |
-| `formatConfig.selector`  | optional | `string`                                                          | `:root`     | The CSS selector containing your CSS custom properties                         |
-| `formatConfig.endOfLine` | optional | `auto` `lf` `crlf` `cr`                                           | `auto`      | [Prettier documentation](https://prettier.io/docs/en/options.html#end-of-line) |
-| `formatConfig.tabWidth`  | optional | `number`                                                          | `2`         | [Prettier documentation](https://prettier.io/docs/en/options.html#tab-width)   |
-| `formatConfig.useTabs`   | optional | `boolean`                                                         | `false`     | [Prettier documentation](https://prettier.io/docs/en/options.html#tabs)        |
+| Parameter                | Required | Type                                                              | Default | Description                                                                    |
+| ------------------------ | -------- | ----------------------------------------------------------------- | ------- | ------------------------------------------------------------------------------ |
+| `formatName`             | optional | `camelCase` `kebabCase` `snakeCase` `pascalCase`                  |         | The case transformation you want to apply to your design token name            |
```

Comment:
You should add a link to the section below

Reply:
Done ✅